### DISCUSSION TITLE
Is this application compatible with Alarmo?

### DISCUSSION THREAD

**Discussion by Geo-Ron** [thanksmister/android-mqtt-alarm-panel#200] [General]
I would like to ask if this application is compatible with Alarmo?
https://github.com/nielsfaber/alarmo

Specifically if it would honour the user codes defined in alarmo...

**Comment by erte1** [comment]
Yes it is.  I use it and pass a code to Alarmo without any problems.

**Reply by array81** [reply]
How did you set up the app? Thanks
